### DISCUSSION TITLE
Units in the extxyz file format for nequip

### DISCUSSION THREAD

**Discussion by Pibemanden** [mir-group/nequip#408] [Q&A]
I am interested in using your code to train a model on some of the data I have collected from ab initio MD.
However they are not in a format that can be read by ase, so I am wondering if there is any conventions for the units used in the extxyz format used by nequip.
I have tried to look into the documentation for both the format and ase, but none of them mention which units are used in the output.

**Comment by Linux-cpp-lisp** [comment]
Hi @Pibemanden ,

Thanks for your interest in our code!

You can use arbitrary units in your data, as long as they are self-consistent. (They **must** be self-consistent:  e.g. if energy is in eV, and position is in Angstrom, forces must be in eV/Angstrom.)  The model's predictions, errors, etc. will be in the units of the provided data.  Datasets with varying units are not supported, they must be converted by the user into one consistent set of units first.

**Reply by Pibemanden** [reply]
Thank you for the quick reply.
This makes it very simple for me to work with the code, however I am still wondering if there is a best practices to follow?
I am sure that coordinates/length units should be Å given the standard for the normal XYZ file. But for the energies do you know if ASE generally uses eV or?
